dashboard/app/models/unit.rb

```diff
@@ -1742,6 +1765,15 @@ def localized_title
     )
   end
 
+  def unit_number
+    has_prefix = unit_group&.has_numbered_units
+
+    return nil unless has_prefix
+
+    position = unit_group_units&.first&.position
+    return position
```

Comment:
```suggestion
    unit_group_units&.first&.position
```

In Ruby the result of the last statement in a method is returned by default, so it's idiomatic to omit the `return` keyword in this case. Also since the `position` variable was only used in the `return` statement we don't really need that either.